src/PkgAuthentication.jl

```diff
@@ -111,6 +111,7 @@ function authenticate(;
         state = initial(server)
         try
             while !(isa(state, Success) || isa(state, Failure))
+                @debug "Calling step(::$(typeof(state)))"
```

Comment:
Unrelated, but I think it's useful to see what state it is in as it goes through its logic.